internal/pkg/cli/app_show.go

```diff
@@ -0,0 +1,312 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"math"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	applicationShowProjectNamePrompt     = "Which project's applications would you like to show?"
+	applicationShowProjectNameHelpPrompt = "A project groups all of your applications together."
+	applicationShowAppNamePrompt         = "Which application of %s would you like to show?"
+	applicationShowAppNameHelpPrompt     = "The detail of an application will be shown (e.g., endpoint URL, CPU, Memory)."
+)
+
+// ShowAppOpts contains the fields to collect for showing an application.
+type ShowAppOpts struct {
+	ShouldOutputJSON bool
+
+	proj *archer.Project
+	app  *archer.Application
+	envs []*archer.Environment
+	uris []*describe.WebAppURI
+
+	appName string
+
+	projectLister archer.ProjectLister
+	projectGetter archer.ProjectGetter
+	appLister     archer.ApplicationLister
+	appGetter     archer.ApplicationGetter
+	envLister     archer.EnvironmentLister
+	identifier    ResourceIdentifier
+
+	w io.Writer
+
+	*GlobalOpts
+}
+
+// BuildAppShowCmd builds the command for showing applications in a project.
+func BuildAppShowCmd() *cobra.Command {
+	opts := ShowAppOpts{
+		w:          os.Stdout,
+		GlobalOpts: NewGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Use:   "show",
+		Short: "Shows all the applications in a project",
```

Comment:
Mmm it doesn't show all of the different applications right?

Maybe `"Displays information about an application per environment."`? (end with a punctuation https://github.com/aws/amazon-ecs-cli-v2/blob/master/STYLE_GUIDE.md#use-full-sentences)

It might be useful to add a Long description too:
```
Displays information about an application per environment.

For Load Balanced Web Applications, displays the URL and path the application can be accessed at.
```